doc/reference/Field.md

```diff
@@ -5,21 +5,21 @@ A field is the representation of a property of an entity.
 * [General Field Settings](#general-field-settings)
```

Comment:
OK, will do.

The links are currently broken on GitHub, see e.g. https://github.com/marmelab/ng-admin/blob/master/doc/reference/Field.md#-date-field-type

They do work on the "GitBook" version, you're right: http://ng-admin-book.marmelab.com/doc/reference/Field.html#-date-field-type

It's a shame that the two differ. I hadn't spotted the "GitBook" version; I'd been referring to the documentation on GItHub directly up until now. If the GitBook version is the official one, it should win, of course.